ui/components/app/modals/account-modal-container/account-modal-container.component.js

```diff
@@ -10,10 +25,52 @@ export default function AccountModalContainer(props, context) {
     showBackButton,
     backButtonAction,
     hideModal,
+    isAccountDetailsModal,
     children,
   } = props;
+  const useBlockie = useSelector((state) => state.metamask.useBlockie);
 
-  return (
+  return process.env.MULTICHAIN && isAccountDetailsModal ? (
+    <Popover
+      className="multichain-account-details"
+      headerProps={{
+        padding: 0,
+        paddingRight: 4,
+        paddingTop: 4,
+        paddingLeft: 4,
```

Comment:
Question: Why are we adding padding and then paddingRight, paddingTop and paddingLeft separately?

Reply:
It doesn't look great, but when I tried to give it paddingBottom:0 , it would not apply. Instead I had to use padding:0 to erase all padding and then reapply the right, top, and left. This is something I'll bring up with the DS team. @garrettbear @georgewrmarshall please let me know if there was a less painful way of doing this

Reply:
I think this is correct?